docs/syscall_descriptions_syntax.md

```diff
@@ -93,6 +93,8 @@ Call attributes are:
 "ignore_return": ignore return value of this syscall in fallback feedback; need to be used for calls
 	that don't return fixed error codes but rather something else (e.g. the current time).
 "breaks_returns": ignore return values of all subsequent calls in the program in fallback feedback (can't be trusted).
+"do_not_generate": do not try to generate this syscall, i.e. use only seed descriptions to produce it.
+"do_not_minimise": do not modify instances of this syscall when trying to minimise a crashing program.
```

Comment:
Nit: we use the `minimize` alternative throughout the project. I think it'd be better to keep it consistent.